.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false
+language: python
+python:
+  - "2.7"
+  - "3.3"
+install:
```

Comment:
I realise none of these versions are pinned. Do the versions change regularly? Presumably happy for the build to break as a result?